src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -899,14 +899,21 @@ private void testResponseRate() {
         feedbacksPage.clickViewResponseLink(courseId, feedbackSessionName);
         feedbacksPage.verifyResponseValue("1 / 1", courseId, feedbackSessionName);
 
+        ______TS("check warning against changing visibility options while editing question with existing response");
```

Comment:
For consistency with the below TS, `check warning is displayed while editing visibility options of question with existing response`.